src/main/java/io/appium/java_client/remote/MobileOptions.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client.remote;
+
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.MutableCapabilities;
+import org.openqa.selenium.ScreenOrientation;
+import org.openqa.selenium.remote.CapabilityType;
+
+import java.net.URL;
+
+public class MobileOptions<T extends MobileOptions<T>> extends MutableCapabilities {
+
+    public MobileOptions() {
+    }
+
+    public MobileOptions(Capabilities source) {
+        this();
```

Comment:
is this call needed?

Reply:
Oh, nope, was a copy paste from the subclass that set the platformName;

Reply:
https://github.com/appium/java-client/pull/1319/commits/499a385e6c067be52b2c8597fdd9d3bf49c547cf